b2b-sample/README.md

```diff
@@ -159,22 +159,12 @@ setup the config json file as mentioned above.
 
 ### Step 6: Setup the identity provider for the Guardio-Business-App
 
-* For this we will create an external identity provider in Asgardeo. After creating an identity provider in Asgardeo.
-
-* Also, On the User Attributes tab, click on + Add User Attributes.
-    * Select `Email`, `First Name`, `Last Name`, and `Username` from the list of attributes.
```

Comment:
This section is not related to this step and moved to the top after the admin app creation.